src/test/rbf_tests.cpp

```diff
@@ -396,6 +396,14 @@ BOOST_FIXTURE_TEST_CASE(improves_feerate, TestChain100Setup)
     // With one more satoshi it does
     BOOST_CHECK(ImprovesFeerateDiagram(pool, {entry1}, {entry1, entry2}, tx1_fee + tx2_fee + 1, tx1_size + tx2_size) == std::nullopt);
 
+    // With prioritisation of in-mempool conflicts, it effects the results of the comparison using the same args as just above
```

Comment:
b380a1df36a95c85310949aeb81d8c57f4d023c4: s/effects/affects

Reply:
RFC: merge the two words into "ffect"

Reply:
done